docs/rest_api.rst

```diff
@@ -177,6 +177,41 @@ Querying Information About Channels and Tokens
    :statuscode 200: Successful query
    :statuscode 500: Internal Raiden node error
 
+.. http:get:: /api/(version)/channels/(token_address)
+
+   Get a list of all unsettled channels for the given token address.
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /api/1/channels/0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8 HTTP/1.1
+      Host: localhost:5001
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      [
+          {
+              "token_network_identifier": "0xE5637F0103794C7e05469A9964E4563089a5E6f2",
+              "channel_identifier": 20,
+              "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
+              "token_address": "0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8",
+              "balance": 25000000,
+              "total_deposit": 35000000,
+              "state": "opened",
+              "settle_timeout": 100,
+              "reveal_timeout": 30
+          }
+      ]
+
+   :statuscode 200: Successful query
```

Comment:
shouldn't it also return some errors here? Just like the next endpoint?

Reply:
No if there are no channels it returns 200 with an empty list just like the plain `/channels` endpoint.

Reply:
What if you give a wrongly formatted address?

Reply:
It should also return a 404.

Reply:
Yes, it's a 400, but still should be listed